src/rock-gazebo.in

```diff
@@ -1,4 +1,14 @@
-#! /bin/sh
+#! /usr/bin/ruby
+require 'rock/bundles'
+require 'optparse'
 
-gazebo --verbose -s @CMAKE_INSTALL_PREFIX@/lib/librock_gazebo.so "$@"
+options = OptionParser.new
+args = options.parse(ARGV)
+world_file = args.shift
 
+Bundles.load
+
+ENV['GAZEBO_MODEL_PATH']=(ENV['GAZEBO_MODEL_PATH']||"")+
```

Comment:
This is a safety net.

This is unlikely, but not impossible. In computer science, it means that it will very certainly happen and that someone will spend a few hours of his time figuring out what the hell. Since the change is very minimal, let's save this person some frustration ;-)